Tests/FunctionalTests/TestApp/Main.js

```diff
@@ -0,0 +1,80 @@
+// process.env['APPLICATION_INSIGHTS_NO_DIAGNOSTIC_CHANNEL'] = true;
```

Comment:
what is this env var doing?

Reply:
This was a helper to locally disable diagnostic channel. Removed!